extensions/skyportal/static/js/components/TNSInfo.jsx

```diff
@@ -9,7 +9,7 @@ const TNSInfo = ({ objID }) => {
   const tnsInfo = useSelector((state) => state.tnsInfo);
 
   useEffect(
-    function fetchTNSInfo() {
```

Comment:
We want these to be named functions actually, please revert

Reply:
Done. [It was pycharm's suggestion.]

Reply:
Ah, actually ESLint complained, so I disabled it.